frontend/src/store/modules/userLists/actions.js

```diff
@@ -3,13 +3,13 @@ import httpService from '../../../services/common/httpService';
 export default {
     getUserLists: (context, userId) => {
         return new Promise((resolve, reject) => {
-            httpService.get('/users/' + userId + '/lists')
-                .then(function (result) {
+            return httpService.get('/users/' + userId + '/lists')
+                .then((result) => {
                     resolve(result.data.data);
                 })
-                .catch(function (error) {
```

Comment:
No need to catch(), save your time. If error occurs, it will be accepted by getUserLists().catch()